advisor/src/main/kotlin/Advisor.kt

```diff
@@ -66,8 +66,12 @@ class Advisor(
             "Read ORT result from '${ortFile.name}' (${ortFile.formatSizeInMib}) in ${duration.inMilliseconds}ms."
         }
 
-        requireNotNull(ortResult.analyzer) {
-            "The provided ORT result file '${ortFile.canonicalPath}' does not contain an analyzer result."
+        if (ortResult.analyzer == null) {
+            log.warn {
+                "The provided ORT result file '${ortFile.canonicalPath}' does not contain an analyzer result."
```

Comment:
Could it be worth mentioning in the warning that this will lead to no vulnerability information being retrieved/ an unchanged OrtResult?

Reply:
Done.